src/api/routes/tasks.ts

```diff
@@ -6,15 +6,55 @@ const router = express.Router();
 // GET request /tasks route
 router.get("/", (req: Request, res: Response, next: NextFunction) => {
   try {
-    console.log(data);
-    res.status(200).json(data);
+    let count: any = req.query.count;
+    // console.log("/-count: ", count);
+    if (count) {
+      let result = [];
+      for (let i: any = 0; i < parseInt(count); i++) {
+        result.push(data.data[i]);
+      }
+      res.status(200).json({ data: result });
+    } else {
+      res.status(200).json(data);
+    }
   } catch (err) {
     //error handling
-    console.log(err);
+    // console.log(err);
     res.status(500).json({
       error: err,
     });
   }
 });
 
+//GET request /tasks/index
+router.get("/:index", (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const idx: number = parseInt(req.params.index);
+    let count: any = req.query.count;
+    // console.log("index: ", idx)
+    // console.log("count: ", count);
+    if (data.data.length > idx) {
+      if (count) {
+        let result = [];
+        const size: number = idx + parseInt(count);
```

Comment:
Watch out for the same problem as above!

Reply:
> > Watch out for the same problem as above!
> 
> In this case, it's returning the data with a possible valid result and filled the rest with null.
> Do I need to return an error message if the count is greater or anything else?

No. If the count exceeds the size, send back the data till the end only.

Reply:
Okay, I will do so.